docs/product/base/typography.html

```diff
@@ -151,15 +151,15 @@
         <div class="stacks-preview--example grid ff-column-nowrap gs16 gsy my0 fs-subheading">
             <div class="grid--cell">
                 <p class="ff-sans mb0">Sans Serif</p>
-                <p class="ff-mono fs-caption fc-light">Arial, "Helvetica Neue", Helvetica, sans-serif</p>
+                <p class="ff-mono fs-caption fc-light">-apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, "Noto Sans", sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji"</p>
```

Comment:
Do we have to reference Segoe at all? I know.. no native typeface for Windows... but given the baseline issue and not a clear way to deal with this in short term could we punt it for now?

Reply:
I’ve considered only adding `-apple-system, BlinkMacSystemFont` to the beginning of our stack as a bit of progressive enhancement for macOS and iOS users. Presumably Windows users would get "Helvetica Neue" if it’s installed, but most users would likely be shown Arial.

...But with that approach, we lose the tidy rationale of simply choosing platform-specific fonts. 🤔